crates/swift-bridge-cli/src/lib.rs

```diff
@@ -1 +1,7 @@
+mod clap_app;
+mod clap_exec;
 
+pub mod app {
```

Comment:
Can we just export a single `run()` function (or some other name) that the `main.rs` calls? 

And in that function we can call `handle_matches(cli().get_matches());`?